### PR TITLE
More python3

### PR DESCRIPTION
**Pull request by mesmith75**

### CODE REVIEW

File: ganga/GangaCore/GPIDev/Lib/Job/Job.py [hunk 1]
```diff
@@ -881,15 +881,19 @@ def createPackedInputSandbox(self, files, master=False):
...
         name = '_input_sandbox_' + self.getFQID('_') + '%s.tgz'
-
         if master:
             if self.master is not None:
                 name = '_input_sandbox_' + self.master.getFQID('_') + '%s.tgz'
             name = name % "_master"
         else:
             name = name % ""
 
-        files = [f for f in files if hasattr(f, 'name') and not f.name.startswith('.nfs')]
```
Comment (mesmith75) [inline ganga/GangaCore/GPIDev/Lib/Job/Job.py:892]: I'm not sure why this line had to be un-condensed but files was empty if now when making the master sandbox. I think it has something to do with `files` being a `map`.

**Comment by mesmith75** [comment]
Now the XML summary merging works - the LHCb scripts are all python 2 so I had to copy them into ganga and convert. I couldn't think of a better solution.

**Comment by mesmith75** [comment]
This can be merged now

**Review by alexanderrichards — APPROVED** [review]

**Comment by alexanderrichards** [comment]
thanks @mesmith75

**Pull request merged**